ios/RCTTWVideoModule.m

```diff
@@ -397,7 +395,18 @@ -(NSMutableDictionary*)convertLocalVideoTrackStats:(TVILocalVideoTrackStats *)st
   }
 }
 
-RCT_EXPORT_METHOD(connect:(NSString *)accessToken roomName:(NSString *)roomName enableAudio:(BOOL *)enableAudio enableVideo:(BOOL *)enableVideo enableData:(BOOL *)enableData encodingParameters:(NSDictionary *)encodingParameters enableNetworkQualityReporting:(BOOL *)enableNetworkQualityReporting dominantSpeakerEnabled:(BOOL *)dominantSpeakerEnabled cameraType:(NSString *)cameraType) {
+RCT_EXPORT_METHOD(connect:(NSString *)accessToken roomName:(NSString *)roomName enableAudio:(BOOL *)enableAudio enableVideo:(BOOL *)enableVideo enableData:(BOOL *)enableData encodingParameters:(NSDictionary *)encodingParameters enableNetworkQualityReporting:(BOOL *)enableNetworkQualityReporting dominantSpeakerEnabled:(BOOL *)dominantSpeakerEnabled cameraType:(NSString *)cameraType)
+{
+  if (encodingParameters[@"fps"]) {
+    fps = [encodingParameters[@"fps"] integerValue];
+  }
+  if(encodingParameters[@"videoWidth"] && encodingParameters[@"videoHeight"]) {
+    int32_t videoWidth = (int32_t)[encodingParameters[@"videoWidth"] integerValue];
+    int32_t videoHeight = (int32_t)[encodingParameters[@"videoHeight"] integerValue];
+    videoDimensions = (CMVideoDimensions){videoWidth, videoHeight};
+  } else {
+    videoDimensions = IPAD ? (CMVideoDimensions){1280, 720} : (CMVideoDimensions){1024, 768};
```

Comment:
@andreychernikovich the different iPad and iPhone resolutions should be set by the JS app calling connect and not internally inside this module